src/Robo/Plugin/Traits/RoboConfigTrait.php

```diff
@@ -15,14 +16,15 @@ trait RoboConfigTrait
      *
      * @param string $key
      *   The key of the configuration to load.
-     *
-     * @return array
-     *   A configuration array.
      */
     protected function getRequiredRoboConfigArrayFor(string $key): array
     {
-        $configValue = $this->getRequiredRoboConfigValueFor($key);
-        $this->validateRoboConfigValueMatchesType($configValue, 'array', $key);
+        $configValue = $this->getRequiredRoboConfigValueFor(key: $key);
+        $this->validateRoboConfigValueMatchesType(
+            configValue: $configValue,
+            expectedType: ConfigTypes::array,
+            key: $key
```

Comment:
```suggestion
            key: $key,
```

Reply:
I see this below too, perhaps what I'm suggesting is wrong?

Reply:
I'm guessing that the trailing comma for named arguments is not valid based upon you resolving this?

Reply:
I committed it! Let me fix up the other instances.

Reply:
Ah, I misread what GitHub was telling me. My bad!